app/assets/stylesheets/topbar.scss

```diff
@@ -1,11 +1,46 @@
 @import 'color';
 
-.menu-text {
-  padding: 0rem 1rem;
-  & a {
-  font-size: 1.2rem;
-    &:hover {
-      color: lighten( $brick, 10% );
+
+.top-bar {
+
+  padding: 0px;
+
+  .top-bar-left {
+    margin-top: 6px;
+  }
+
+
+  .menu-text {
+    margin-top: 1px;
+    a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3